<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-VQS1UIG" failures="1" tests="4" name="Test" time="12.23" errors="0" timestamp="2022-07-14T12:02:15 CEST">
  <testcase classname="soap.MultipleTestSuite" name="SoapTest" time="8.416">
    <failure type="java.lang.AssertionError" message="expected [FAILED] but found [FINISHED]">
      <![CDATA[java.lang.AssertionError: expected [FAILED] but found [FINISHED]
at soap.MultipleTestSuite.SoapTest(MultipleTestSuite.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- SoapTest -->
  <testcase classname="Rest.restcountriesComTestSuite" name="SoapTest" time="1.398"/>
  <testcase classname="soap.testSeperatly" name="singleTest" time="0.568"/>
  <testcase classname="soap.FirstTest" name="SoapTest" time="1.799"/>
</testsuite> <!-- Test -->
